llvm/lib/Target/AMDGPU/SIFrameLowering.cpp

```diff
@@ -678,6 +678,12 @@ void SIFrameLowering::emitEntryFunctionPrologue(MachineFunction &MF,
         break;
       }
     }
+
+    // FIXME: We can spill incoming arguments and restore at the end of the
+    // prolog.
+    if (!ScratchWaveOffsetReg)
```

Comment:
Is this change related?

Reply:
Kind of. One test hit the assert due to the sgpr limiting attribute